components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m

```diff
@@ -218,6 +218,7 @@ - (void)testSnackbarOverlayViewWithHighElevation {
   MDCSnackbarMessageView *messageView = [self snackbarMessageViewWithText:kItemTitleShort1Latin
                                                               actionTitle:kItemTitleShort2Latin];
   messageView.frame = CGRectMake(0, 0, kWidth, kHeightSingleLineText);
+  self.testManager.internalManager.overlayView.frame = CGRectMake(0, 0, 375, 667);
```

Comment:
Tip: File an Issue to investigate why this is different on Xcode/Bazel builds and add a TODO here linking to that Issue.

Reply:
Done.